### PR TITLE
Improve UI  for Country Currency Settings Fixes #704

### LINKED ISSUE #704: Improve UI to reduce confusion about Currency & Country settings
This is quite likely one of the top 5 gotchas for new users. What happens is they'll try to change currency settings on the Event Espresso > General Settings > Countries tab, and then not see any change to the currency displayed. This is because they have not changed the country set under Organization settings.

Here are two things that will help:
1) Add help text just under the Country field that communicates there is a relationship.
screen shot 2018-09-24 at 10 13 44 am

2) Disable all Currency setting inputs for every country except for the Country that's set in General Setings > Organization Settings. 
screen shot 2018-09-24 at 10 06 39 am
It may also help to include a note in the help text that says which country is currently selected under Organization settings.

### PR DESCRIPTION
**Pull request by tn3rb**
## Problem this Pull Request solves
Fixes #704 

Implemented the following UI improvements as requested in the connected issue:

- added help text just under the Country field that communicates there is a relationship.
- disabled all Currency setting inputs for every country except for the Country that's set in General Settings > Organization Settings.
- included some help text regarding the Currency setting inputs that says which country is currently selected under Organization settings.

## How has this been tested
tested via browser usage only

## Checklist

* [ ] I have added a changelog entry for this pull request
* [x] I have read the documentation relating to systems affected by this pull request, see https://github.com/eventespresso/event-espresso-core/tree/master/docs
* [x] User input is adequately validated and sanitized
* [x] all publicly displayed strings are internationalized (usually using `esc_html__()`, see https://codex.wordpress.org/I18n_for_WordPress_Developers)
* [x] My code is tested.
* [x] My code follows the Event Espresso code style.
* [x] My code has proper inline documentation.
* [x] My code accounts for when the site is Maintenance Mode (MM2 especially disallows usage of models)

### CODE REVIEW

File: admin_pages/general_settings/templates/country_details_settings.template.php [hunk 1]
```diff
@@ -7,8 +14,24 @@
...
+                                'event_espresso'
+                            ),
+                            $CNT_name_for_site
+                        ); ?>
+                    </span>
+                </p>
+            <?php } ?>
+            <br/>
+            <input id="country_settings_save_2" class="button-primary save" type="submit" name="save"
+                        value="<?php _e('Save Country Details', 'event_espresso'); ?>">
```
Comment (nerrad) [inline admin_pages/general_settings/templates/country_details_settings.template.php]: Should this be `esc_html_e`?

**Comment by tn3rb** [comment]
> I wonder if consideration has been given to what happens if someone changes their country in the organization settings but does not visit the country settings page to make sure the currency stuff is setup the way they want?

that can certainly be discussed on the original issue, or via a new issue, or can wait to see if any users actually report that happening before making additional changes.

**Comment by tn3rb** [comment]
### testing

- visit the **General Settings** > **Organization Settings** admin tab
- view the help text just under the Country field
- [ ] confirm that no typos and/or grammatical issues exist
- take note of the country set on that page, then click on the **Countries** tab
- check that the settings for your site's country are loaded
- scroll down to the Currency setting inputs
- [ ] confirm that the inputs are enabled and can be edited (ie: change and submit)
- now choose a different country from the select input at the top of the page
- scroll down to the Currency setting inputs
- [ ] confirm that the inputs are now disabled and cannot be edited
- help text should appear above the submit button for the country settings
- [ ] confirm that no typos and/or grammatical issues exist
- feel free to experiment with additional testing that may reveal issues with the work in this branch
- plz compare any potential issues against master branch to determine if they are caused by the work in this branch

**Comment by joshfeck** [comment]
The UI changes work great. One minor improvement would be if the "Currency setting inputs are currently only enabled" text could be moved before the inputs. This way the user will see the explanation why the form inputs are disabled, without having to scroll down the page. Could the next help text be moved so it appears right after the first paragraph before the inputs?

**Comment by sethshoultes** [comment]
@tn3rb can you please make the changes Josh mentioned?

**Comment by tn3rb** [comment]
> One minor improvement would be if the "Currency setting inputs are currently only enabled" text could be moved before the inputs. 

do you mean just before the currency setting inputs (ie: after the row with the "Country Name" label and input) or before ALL of the inputs (ie: at the top of the page after "update your Contact Information on the Your Organization tab.") ?

> Could the next help text be moved so it appears right after the first paragraph before the inputs?

sry but what is the "next help text" ? is that the same thing as the "Currency setting inputs are currently only enabled" text, or something else?

**Comment by joshfeck** [comment]
Sorry, I mistyped, or autocorrect mangled what I meant to type. I meant to type "the new help text". So basically put all the help text within one area on the page.

screenshot_2018-10-01 testing machine it s ok wordpress

**Comment by tn3rb** [comment]
### testing round 2

- visit the **General Settings** > **Organization Settings** admin tab
- take note of the country set on that page, then click on the **Countries** tab
- check that the settings for your site's country are loaded
- help text should not include anything pertaining to the currency settings being disabled
- now choose a different country from the select input at the top of the page
- help text should appear below the existing text at the top of the page
- [ ] confirm that no typos and/or grammatical issues exist
- feel free to experiment with additional testing that may reveal issues with the work in this branch
- plz compare any potential issues against master branch to determine if they are caused by the work in this branch

**Comment by joshfeck** [comment]
I fixed a typo so after that change is reviewed this is ready for merging.

**Pull request merged**